sections/04_study.md

```diff
@@ -560,31 +561,33 @@ metabolic — is key if precision medicine is to be targeted not only to a
 specific individual, but also to specific pathological subsets of cells.
 Single-cell methods also promise to uncover a wealth of new biological
 knowledge. A sufficiently large population of single cells will have enough
-representative “snapshots” to recreate timelines of rapid biological processes.
-If tracking processes over time is not the limiting factor, single cell
+representative "snapshots" to recreate timelines of dynamic biological processes.
+If tracking processes over time is not the limiting factor, single-cell
 techniques can provide maximal resolution compared to averaging across all cells
 in bulk tissue, enabling the study of transcriptional bursting with single-cell
 FISH or the heterogeneity of epigenetic patterns with single-cell Hi-C or
-ATAC-seq [@tag:Liu2016_sc_transcriptome @tag:Vera2016_sc_analysis].
+ATAC-seq [@tag:Liu2016_sc_transcriptome @tag:Vera2016_sc_analysis].  Joint
+profiling of single-cell epigenetic and transcriptional states provides
+unprecedented views of regulatory processes [@doi:10.1101/138685].
 
 However, large challenges exist in studying single cells. Relatively few cells
 can be assayed at once using current droplet, imaging, or microwell
 technologies, and low-abundance molecules or modifications may not be detected
-by chance in a phenomenon known as dropout. To solve this problem, Angermueller
+by chance due to a phenomenon known as dropout. To solve this problem, Angermueller
 et al. [@tag:Angermueller2016_single_methyl] trained a neural network to predict
 the presence or absence of methylation of a specific CpG site in single cells
 based on surrounding methylation signal and underlying DNA sequence, achieving
 several percentage points of improvement compared to random forests or deep
 networks trained only on CpG or sequence information. Similar deep learning
 methods have been applied to impute low-resolution ChIP-seq signal from bulk
-tissue with great success, and they could easily be adapted to single cell data
+tissue with great success, and they could easily be adapted to single-cell data
 [@tag:Qin2017_onehot @tag:Koh2016_denoising]. Deep learning has also been useful
 for dealing with batch effects [@tag:Shaham2016_batch_effects].
 
 Examining populations of single cells can reveal biologically meaningful subsets
 of cells as well as their underlying gene regulatory networks
 [@tag:Gaublomme2015_th17]. Unfortunately, machine learning generally struggles
-with unbalanced data — when there are many more inputs of class 1 than class 2 —
+with imbalanced data — when there are many more examples of class 1 than class 2 —
```

Comment:
This is inconsistent throughout.  It's on my TODO list for the manuscript-wide consistency editing, along with abbreviations.